Enable Customizing imageResizeUpscale

yob-yob opened the pull request:
This will allow devs to disable image upscaling when using the FilePond ImageResize Feature.

I need this because I was literally lost on why users uploading 3MB image are getting errors saying that the image they uploaded exceeds 10MB, and after 500 years (2 hours) of debugging I found out it's because of this.

Please accept this 😭 

Should I create tests for this be accepted?

yob-yob committed:
Enable Customizing imageResizeUpscale

yob-yob committed:
Fix: Access to an undefined property

Fix PhpStan Error: Access to an undefined property Filament\Forms\Components\FileUpload::$imageResizeUpscale.

yob-yob committed:
chore: fix code style

danharrin commented:
I've checked the Filepond docs, should this be a bool instead of a string setting?

yob-yob commented:
sorry about that.. I'll change it to bool...

yob-yob committed:
Updates

- Change Parameter Type from String to Bool
- Fix: imageResizeUpscale filepond option has no ending comma

danharrin commented:
Have you tested this?

danharrin commented on packages/forms/src/Components/FileUpload.php:
```diff
@@ -184,6 +193,11 @@ public function getImageResizeMode(): ?string
         return $this->evaluate($this->imageResizeMode);
     }
 
+    public function getImageResizeUpscale(): ?string
```
because this doesnt look right?

yob-yob replied:
Sorry about that @danharrin... I change it to `bool`

yob-yob committed:
Fix Getter for ImageResizeUpscale

danharrin committed:
Update FileUpload.php

danharrin committed:
Update FileUpload.php
